src/main/java/task/Todo.java

```diff
@@ -0,0 +1,44 @@
+package task;
+
+import java.time.LocalDateTime;
+import java.io.Serializable;
+
+/**
+ * This extension of the task class will allow the user to add a task of to-do
+ * type.
+ *
+ * @author Sai Ganesh Suresh
+ * @version v2.0
+ */
+public class Todo extends Task implements Serializable {
+
+    public Todo(String description) {
```

Comment:
Same here. If Todo does not have any specialized behavior compare to Task, then what is the purpose of having this Class?